examples/http-example/main.go

```diff
@@ -26,6 +26,15 @@ var handler = http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 })
 
 func main() {
+	// uncomment the below to use the caching key provider
+	// u, err := url.Parse("https://<your-domain>")
+	// if err != nil {
+	// 	// we'll panic in order to fail fast
+	// 	panic(err)
+	// }
+
+	// p := josev2.NewCachingJWKSProvider(*u, 5*time.Minute)
+
```

Comment:
Why not making this the default example, and fixed secret as alternative?  JWKS is more standard approach.

Reply:
It definitely is, but to get the example working you then need to have an Auth0 account. I liked that this example worked without an account. Maybe it would be better to break it off into a separate example so we have one example without JWKS and one example with JWKS?

Reply:
Separate examples would be nice.